src/shared/components/mutationTable/column/styles.module.scss

```diff
@@ -0,0 +1,9 @@
+.cosmic-count {
```

Comment:
I thought since theres not much duplication it would be better to decouple them in case anything changes. Not opposed to unifying them if you think thats better.